### ISSUE TITLE
insall problem openvz

### ISSUE THREAD

**Issue by yozk** [StreisandEffect/streisand#1380]
Hello!

Streisand vpn script give's me error, when i try install it to openvz vps in localhost mode.
Isn't supported openvz? Or i do something wrong?

`
TASK [ip-forwarding : Enable IPv4 traffic forwarding] ***************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to reload sysctl: net.ipv4.tcp_syncookies = 1\nkernel.msgmnb = 65536\nkernel.msgmax = 65536\nkernel.shmmax = 68719476736\nkernel.shmall = 4294967296\nnet.ipv4.conf.all.accept_source_route = 0\nnet.ipv4.conf.default.accept_source_route = 0\nnet.ipv4.conf.all.log_martians = 1\nnet.ipv4.conf.default.log_martians = 1\nnet.ipv4.conf.all.accept_redirects = 0\nnet.ipv4.conf.default.accept_redirects = 0\nnet.ipv4.conf.all.send_redirects = 0\nnet.ipv4.conf.default.send_redirects = 0\nnet.ipv4.conf.all.rp_filter = 0\nnet.ipv4.conf.default.rp_filter = 0\nnet.ipv4.icmp_echo_ignore_broadcasts = 1\nnet.ipv4.icmp_ignore_bogus_error_responses = 1\nnet.ipv4.conf.all.secure_redirects = 0\nnet.ipv4.conf.default.secure_redirects = 0\nkernel.randomize_va_space = 1\nnet.ipv4.ip_forward = 1\nsysctl: permission denied on key 'kernel.sysrq'\nsysctl: permission denied on key 'kernel.core_uses_pid'\nsysctl: permission denied on key 'net.core.wmem_max'\nsysctl: permission denied on key 'net.core.rmem_max'\nsysctl: permission denied on key 'fs.suid_dumpable'\nsysctl: cannot stat /proc/sys/fs/protected_hardlinks: No such file or directory\nsysctl: cannot stat /proc/sys/fs/protected_symlinks: No such file or directory\n"}

RUNNING HANDLER [ssh : Restart SSH] *********************************************************************************************************************************
        to retry, use: --limit @/root/streisand/playbooks/localhost.retry

PLAY RECAP **********************************************************************************************************************************************************
localhost                  : ok=50   changed=30   unreachable=0    failed=1
`

**Comment by yozk** [comment]
Hello @cpu !

Streisand work's fine on kvm or xen vps?

**Comment by cpu** [comment]
@yozk Yes, those should both work fine. For the best results I recommend using one of the providers that Streisand explicitly supports in the setup: DigitalOcean, Linode, AWS, GCE, Rackspace, etc.
